### PR TITLE
docs: refactor huawei cloud doc

### PR DESCRIPTION
**Pull request by Gala-R**

### CODE REVIEW

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 1]
```diff
@@ -1,163 +1,164 @@
...
+          storageClassName: csi-disk
+          resources:
+            requests:
+              storage: 20Mi
+          accessModes:
+            - ReadWriteOnce
+    dashboardServiceTemplate:
+      metadata:
+        annotations:
+          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0252.html#cce_10_0252__table268711532210
```
Comment (Rory-Z) [inline docs/zh_CN/deployment/on-huawei-cloud.md:118]:
```suggestion
          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html#section4
```

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 2]
```diff
@@ -1,163 +1,164 @@
...
+    template:
+      spec:
+        emqxContainer:
+          image:
+            repository: emqx/emqx-ee
+            version: 4.4.14
+    serviceTemplate:
+      metadata:
+        annotations:
+          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0252.html#cce_10_0252__table268711532210
```
Comment (Rory-Z) [inline docs/zh_CN/deployment/on-huawei-cloud.md:60]:
```suggestion
          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html#section4
```

Comment (Rory-Z) [reply docs/zh_CN/deployment/on-huawei-cloud.md:60]: Check this document，looks the `kubernetes.io/elb.class` is request ?

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 3]
```diff
@@ -1,163 +1,164 @@
...
 
-> Kubernetes 集群节点必须可以访问外网（可以通过加 NAT 网关解决）
+  - 通过本地安装 kubectl 工具连接 CCE 集群：具体请参考：[使用 kubectl 连接集群](https://support.huaweicloud.com/usermanual-cce/cce_10_0107.html#section3)
+  - 通过 CloudShell 连接 CCE 集群：具体请参考：[使用 CloudShell 连接集群](https://support.huaweicloud.com/usermanual-cce/cce_10_0107.html#section2)
 
-> Kubernetes 集群节点的操作系统建议是 Ubuntu，否则有可能会缺少必要的库（socat）
+- 安装 EMQX Operator：具体请参考：[安装 EMQX Operator](../getting-started/getting-started.md)
 
-## 为 EMQX 集群配置持久化存储
+> Kubernetes 集群节点必须可以访问外网（可以通过加 NAT 网关解决）
```
[the remark this answers is not in the document]
Comment (Gala-R) [reply docs/zh_CN/deployment/on-huawei-cloud.md:20]: > 这个是必须的么？如果不能访问外网，会发生什么？

外部镜像拉不下来

File: docs/zh_CN/deployment/on-huawei-cloud.md (hunk 2 above)
Comment (Gala-R) [reply docs/zh_CN/deployment/on-huawei-cloud.md:60]: > Check this document，looks the `kubernetes.io/elb.class` is request ?

Not necessary, I verified

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 4]
```diff
@@ -1,204 +1,215 @@
...
+云容器引擎深度整合高性能的计算（ECS/BMS）、网络（VPC/EIP/ELB）、存储（EVS/OBS/SFS）等服务，并支持 GPU、NPU、ARM 等异构计算架构，支持多可用区（Available Zone，简称 AZ）、多区域（Region）容灾等技术构建高可用 Kubernetes 集群。关于更多云容器引擎 CCE 产品介绍，请查看 [什么是云容器引擎](https://support.huaweicloud.com/productdesc-cce/cce_productdesc_0001.html?utm_source=cce_Growth_map&utm_medium=display&utm_campaign=help_center&utm_content=Growth_map)。
 
-华为云是全球首批 Kubernetes 认证服务提供商（Kubernetes Certified Service Provider，KCSP），是国内最早投入 Kubernetes 社区的厂商，是容器开源社区主要贡献者和容器生态领导者。华为云也是 CNCF 云原生计算基金会的创始成员及白金会员，云容器引擎是全球首批通过 CNCF 基金会 Kubernetes 一致性认证的容器服务。
+## 前提条件
 
-关于更多云容器引擎 CCE 产品介绍，请查看 [什么是云容器引擎](https://support.huaweicloud.com/productdesc-cce/cce_productdesc_0001.html?utm_source=cce_Growth_map&utm_medium=display&utm_campaign=help_center&utm_content=Growth_map)。
+在开始之前，你需要准备以下内容：
 
-## 前提条件
+- 开通华为云容器服务，并创建一个 CCE 集群。具体请参考：[创建 CCE 集群](https://support.huaweicloud.com/usermanual-cce/cce_01_0028.html)
```
Comment (Rory-Z) [inline docs/zh_CN/deployment/on-huawei-cloud.md:11]:
```suggestion
- 开通华为云容器服务，并创建一个 CCE 集群。具体请参考：[创建 CCE 集群](https://support.huaweicloud.com/usermanual-cce/cce_01_0028.html)
    ::: tip
    Kubernetes 集群节点必须可以访问外网（可以通过加 NAT 网关解决），否则无法拉取除容器镜像服务（SoftWare Repository）外的第三方镜像
    :::
    
    :::tip
    Kubernetes 集群节点的操作系统建议选择 Ubuntu，否则有可能会缺少必要的库（socat）
    :::
```

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 5]
```diff
@@ -1,204 +1,215 @@
...
 
-> Kubernetes 集群节点必须可以访问外网（可以通过加 NAT 网关解决）
+  - 通过本地安装 kubectl 工具连接 CCE 集群：具体请参考：[使用 kubectl 连接集群](https://support.huaweicloud.com/usermanual-cce/cce_10_0107.html#section3)
+  - 通过 CloudShell 连接 CCE 集群：具体请参考：[使用 CloudShell 连接集群](https://support.huaweicloud.com/usermanual-cce/cce_10_0107.html#section2)
 
-> Kubernetes 集群节点的操作系统建议是 Ubuntu，否则有可能会缺少必要的库（socat）
+- 安装 EMQX Operator：具体请参考：[安装 EMQX Operator](../getting-started/getting-started.md)
 
-## 为 EMQX 集群配置持久化存储
+> Kubernetes 集群节点必须可以访问外网（可以通过加 NAT 网关解决）
```
Comment (Rory-Z) [inline docs/zh_CN/deployment/on-huawei-cloud.md:20]: Move it to after 11 line

File: docs/zh_CN/deployment/on-huawei-cloud.md [hunk 6]
```diff
@@ -1,163 +1,164 @@
...
+              "bandwidth_size": 5,
+              "bandwidth_sharetype": "PER",
+              "eip_type": "5_bgp"
+            }
+      spec:
+        type: LoadBalancer
+    listenersServiceTemplate:
+      metadata:
+        annotations:
+          kubernetes.io/elb.autocreate: |
```
[the remark this answers is not in the document]
Comment (Rory-Z) [reply docs/zh_CN/deployment/on-huawei-cloud.md:132]: https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html#cce_10_0014__table939522754617

[the remark this answers is not in the document]
Comment (Gala-R) [reply docs/zh_CN/deployment/on-huawei-cloud.md:137]: > https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html#cce_10_0014__table939522754617

done

File: docs/zh_CN/deployment/on-huawei-cloud.md (hunk 4 above)
Comment (Gala-R) [reply docs/zh_CN/deployment/on-huawei-cloud.md:11]: > 

Done

File: docs/zh_CN/deployment/on-huawei-cloud.md (hunk 5 above)
Comment (Gala-R) [reply docs/zh_CN/deployment/on-huawei-cloud.md]: > Move it to after 11 line

Done

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**